src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/eventlog/EventLogService.kt

```diff
@@ -9,13 +8,7 @@ import uk.gov.justice.digital.hmpps.personrecord.service.cprdomainevents.events.
 @Component
 class EventLogService(
   private val eventLogRepository: EventLogRepository,
-  private val objectMapper: ObjectMapper,
 ) {
 
-  fun logEvent(eventLog: RecordEventLog): EventLogEntity = eventLogRepository.save(
-    EventLogEntity.from(
-      eventLog,
-      eventLog.clusterComposition?.let { objectMapper.writeValueAsString(eventLog.clusterComposition) },
-    ),
-  )
+  fun logEvent(eventLog: RecordEventLog): EventLogEntity = eventLogRepository.save(EventLogEntity.from(eventLog))
```

Comment:
This is a lazy class now.
This is a bit childish but sums up the term:
https://refactoring.guru/smells/lazy-class